components/places/sql/create_shared_triggers.sql

```diff
@@ -61,7 +61,10 @@ BEGIN
     WHERE NEW.guid = 'root________'
       AND NEW.parent IS NOT NULL;
 
-    SELECT throw('update: item without parent')
+    SELECT throw(format(
+        'update: item without parent: operation=%q',
+        CASE WHEN NEW.syncChangeCounter > 0 THEN 'sync' ELSE 'user' END
```

Comment:
I'm actually not sure we can entirely trust this `operation` - the sync change counter might be > 0 even for user operations (eg, on a second operation done to the bookmark before syncing). But this seems fine, so long as we understand this.